### DISCUSSION TITLE
OpenFast AG04MOD dat file

### DISCUSSION THREAD

**Discussion by Oisin-C** [OpenFAST/openfast#1286] [General]
Hello,
Does anyone know if a .dat file exists for a type of turbine blade geometry by the name of "AG04MOD"? Thanks in advance

**Comment by RBergua** [comment]
Hi Oisin,

We are using the airfoil AG04MOD in the OC6 Phase IV project. I believe you are a participant of this project. The information is available in the Appendix B of the report: "Definition of the SOT TetraSpar Floating Wind System for OC6 Phase IV". This report is available in the Confluence site.
![image](https://user-images.githubusercontent.com/33709140/195877303-24f1a056-9ff3-4726-98a3-c8dafa2eba09.png)
If necessary, the airfoil shape is also available in the same appendix.

I hope that helps!

Roger
